src/utils/sentry.js

```
@@ -1,6 +1,6 @@
-import * as Sentry from 'sentry-expo'
+import * as Sentry from '@sentry/react-native'
```

@alitokgonul please double check that all Sentry setup in this file works for the web environment.

---

The React Native Sentry SDK (@sentry/react-native) is not compatible with web platforms when running Expo apps in the browser. While the SDK works correctly on iOS and Android devices, it silently fails on the web due to missing native modules and platform-specific dependencies.
If we want to support Sentry on the web, we need to use @sentry/browser. Let's create a new card for this.


BTW, I successfully sent errors to Sentry on the web by manually crafting and sending the payload using fetch, without relying on the official Sentry SDK configuration. This approach bypasses @sentry/react-native and doesn’t use the typical Sentry.init() setup for web, but it works as a temporary workaround. 
https://atxanka.sentry.io/issues/6833291539/?alert_rule_id=15815901&alert_type=issue&notification_uuid=cf20cd2a-bb1a-48b7-8b26-bf1da306b6e1&project=4508880446423040&referrer=slack

---

I am happy to move web support to a new ticket, but it will still need to be done before we merge this to `bm-develop` as web is an important platform for this application.
I have created biblemesh/ereader-development#52 for this.